torch/csrc/jit/passes/symbolic_shape_runtime_fusion.cpp

```diff
@@ -163,7 +163,8 @@ StrideInput summarizeOutputStrides(const TensorType& tt) {
   // We only try to maintain output striding for channels last tensors,
   // otherwise we defer to contiguous
   // TODO: channels last 3d
-  if (c10::is_channels_last_strides_2d(sizes, strides)) {
+  // NNC Channels last permutation for outputs causes slowdown, disable
```

Comment:
Is this case still slow after the perf improvements we did?

Reply:
yea, for cpu it is